src/utils/utils.ts

```diff
@@ -1,3 +1,30 @@
 export function format(first: string, middle: string, last: string): string {
-  return (first || '') + (middle ? ` ${middle}` : '') + (last ? ` ${last}` : '');
+  return (
+    (first || '') + (middle ? ` ${middle}` : '') + (last ? ` ${last}` : '')
+  );
+}
+
+/**
+ * Get all of the slotted children in the root element that match `nodeName`
+ */
+export function getSlottedNodes(
+  root: HTMLElement,
+  nodeName: string,
+): Array<Node> {
```

Comment:
I'll try to spend a few more minutes and see if I can get a test to work this time, but I made some local changes just to confirm that this is an integral part of the component workings, and a bunch of tests failed when I changed the return value of this function. I'll spare you most of the verbose output, but this is what I got:

![image](https://user-images.githubusercontent.com/2008881/121432603-bc9d3f80-c92f-11eb-971b-2125f6cff43f.png)


When I made these changes:

![image](https://user-images.githubusercontent.com/2008881/121432663-d0e13c80-c92f-11eb-8dec-5c376639f2a0.png)

Reply:
Also while I (sort of) remember, I'll put this here in case I want to come back to it later when referring to why this was done. When I wrote this function originally as a helper for `<va-accordion>`, I couldn't unit test it because the mock object was missing some important things. It didn't seem worth the effort to manually mock out _everything_ required since what was provided was pretty bare.